src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +548,49 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_get_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        # Test based on given range of Zeiss stage
+        dependency = simulated.Stage("stage", "test", axes=["y", "m"],
+                                     ranges={"y": [0.e-3, 100.e-3], "m": [0.e-3, 13.e-3]})
+
+        # Transform into Meteor stage , rotated by 26 degrees --> 0.4537856055185 rad
+        tilt = math.radians(26)
+        stage = ConvertStage("conv", "stage", {"orig": dependency},
+                             axes=["y", "m"], rotation=tilt)
+        first_value = 2e-03
+        second_value = 0
+        f = stage.moveAbs({"x": first_value, "y": second_value})
+        f.result()
+
+        testing.assert_pos_almost_equal(stage.position.value, {"x": first_value, "y": second_value})
+        y_value = first_value * math.cos(tilt)
+        m_value = first_value * math.sin(tilt)
+        testing.assert_pos_almost_equal(dependency.position.value, {"y": y_value, "m": m_value})
+        # Move the stage in FM imaging range at different points
+        # and check if ConvertStage coordinates fall in between the ConvertStage range
+        try:
+            f = dependency.moveAbs({"y": 30.e-3, "m": 0.e-3})
+            f.result()
+        except:
+            self.fail('Riased unexpected exception')
```

Comment:
No need to catch every exception and convert them to a failure. The unitest framework already does this.